### DISCUSSION TITLE
curl nsclient++

### DISCUSSION THREAD

**Discussion by KoenHaemels** [mickem/nscp#813] [General]
Hi

I would like to inform if nsclient++ is impacted by the curl vulnerability: https://www.rezilion.com/blog/cve-2023-38545-a-high-severity-curl-and-libcurl-cve-to-be-published-on-october-11th/

Is there any info if the product is impacted?

Kind regards

**Comment by KoenHaemels** [comment]
@mickem

**Comment by mickem** [comment]
Sorry, I missed this, NSClient++ it self does not use curl so is not vulnerable to this.
